src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
```

Comment:
Please add UNIQUE constraint or use SERIAL that is an alias for BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE according to https://dev.mysql.com/doc/refman/8.3/en/numeric-type-syntax.html